NEW_MODULE_TECH_EVAL.MD

```diff
@@ -41,7 +41,8 @@ Information gathered includes:
 * Link to self evaluation results
 * Informational Section - provides info, instead of eliciting it.
   * A link/pointer to the most recent version of the Acceptance Criteria.
-  * A maximum duration (3 weeks) from the submission date for the initial review.  
+  * The TC has a maximum duration of 3 weeks to perform the initial review.
+    * The evaluator may have between 2 and 3 weeks, depending when the TC meeting happens during that first week.  
```

Comment:
@mweaver-ebsco pointed out this text was ambiguous before.  @starsplatter @craigmcnally and I agreed.  Does this help?

Reply:
Somehow making it more ambiguous with 2-3 weeks actually helps make it less ambiguous 😄 I like it!